echidna/limit/limit_setting.py

```diff
@@ -209,7 +270,8 @@ def _get_chi_squared(self, backgrounds, total_backgrounds, current=-1):
                         self._calculator.get_chi_squared(
                             self._observed,
                             expected,
-                            penalty_term=penalty_term))
+                            penalty_term=penalty_term),
+                        count, background.sum())
```

Comment:
I used hanging indents as PEP8 suggests https://www.python.org/dev/peps/pep-0008/#indentation, as the first argument is another function call, with arguments, it makes it easier to read.